coding-concepts.js

```diff
@@ -10,37 +10,44 @@
 const cohort = "Echo 2022"
 // console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It will split each individual character and have a comma in between them. For example: E,c,h,o .... and so on.
+// b) Verify and explain:[
+//   'E', 'c', 'h',
+//   'o', ' ', '2',
+//   '0', '2', '2'
+// ]
+
+// I was correct in the fact that it split each value and seperated them by a comma but I was incorrect because it put them into an array.
```

Comment:
yes, the split method changes a string into an array as well as the separation aspect you addressed